aml-development/frontend
# feat(listing): API Listing

Adds default icon and banner images (Front end only) for the API listing
Adds the createEdit functionality
Removes screenshots from API listing page

Closes #AMLOS-766, 698

**Description**
An API listing with look slightly different on AML.  The create/edit page needs an update so that when a user selects to create an API some fields are hidden (graphics), and new fields are shown (schema/etc)

**Acceptance Criteria**:
A user can create an API listing from the Create/Edit page.

There may need to be further improvements made to listing view so that user can better understand API listings.

**how to test**
pull this branch and https://github.com/aml-development/ozp-backend/pull/450
you should see 2 new listings, both with the API banner.
You should be able to go into create edit and make a new API listing. You should not be required to add graphics.
